internal/backends/qt/qt_window.rs

```diff
@@ -716,6 +716,12 @@ impl ItemRenderer for QtItemRenderer<'_> {
             LineCap::Round => 0x20,
             LineCap::Square => 0x10,
         };
+        let stroke_pen_join_style: i32 = match path.stroke_line_join() {
+            LineJoin::Miter => 0x00,
+            LineJoin::Round => 0x80,
+            LineJoin::Bevel => 0x40,
+        };
+
```

Comment:
Not an issue for this PR, but I'm not a fan of these magic numbers and we should replace them with constants